code/xcvm/lib/core/src/transport/ibc/mod.rs

```diff
@@ -84,19 +96,30 @@ pub fn to_cw_message<T>(
 				timeout: route.counterparty_timeout.absolute(block),
 				memo: Some(memo),
 			};
-			Ok(WasmMsg::Execute {
-				contract_addr: addr.into(),
-				msg: to_binary(&transfer)?,
-				funds: <_>::default(),
-			}
-			.into())
+			Ok((
+				WasmMsg::Execute {
+					contract_addr: addr.into(),
+					msg: to_binary(&transfer)?,
+					funds: <_>::default(),
```

Comment:
why is it correct to pass a default here?

Reply:
because precompile impl maps coing to pallet-assets and transfers from sender. no need to send 2 times. btw same with cosmos